.github/workflows/npm-brew.yaml

```diff
@@ -67,3 +67,52 @@ jobs:
         env:
           HOMEBREW_GITHUB_API_TOKEN: ${{secrets.HOMEBREW_GITHUB_API_TOKEN}}
         run: ./ci/steps/brew-bump.sh
+
+  aur:
+    needs: npm
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    env:
+      GH_TOKEN: ${{ secrets.HOMEBREW_GITHUB_API_TOKEN }}
```

Comment:
I feel like we should rename the secret to `CDRCI_GITHUB_API_TOKEN` if we're going to use it in both places. I'll have to find the value or ask @deansheather again though if I do.

Reply:
Makes sense.  For now this seems chill.